api/server/appsettings.Development.json

```diff
@@ -45,5 +45,23 @@
         "Name": "Console"
       }
     ]
+  },
```

Comment:
maybe copy this config to appsettings.json too, so we have some default values set up.

Reply:
Okay, I understand. btw, should I still separate the configuration files like in v1? Or do we just need to keep the options in appsettings for now?

Reply:
@iammukeshm i added default value to appsettings